server/tests/common/account.py

```diff
@@ -40,35 +41,56 @@
 BOB_ACCOUNT_CREATED_ON = DateTime(2000, 1, 2)
 
 
+async def create_account_with_password(
+    account_component: MemoryAccountComponent,
+    account_email: EmailAddress,
+    auth_method_id: AccountAuthMethodID,
+    created_by_ip: str,
+    created_by_user_agent: str,
+    created_on: DateTime,
+    human_label: str,
+    mac_key: SecretKey,
+    password_secret_algorithm: PasswordAlgorithmArgon2ID,
+    vault_key_access: bytes,
+):
+    assert account_email not in account_component._data.accounts
+    email_token = EmailValidationToken.new()
+    account_component._data.unverified_emails[account_email] = (email_token, created_on)
```

Comment:
There is still a TODO to put here since this code won't work with the PostgreSQL implementation